docs/sources/tempo/shared/tempo-in-grafana.md

```diff
@@ -44,6 +44,21 @@ Use the Explore trace view to quickly diagnose errors and high latency events in
 
 ![Sample search visualization](/static/img/docs/grafana-cloud/trace_search.png)
 
+### Search is non-deterministic
+
+Most search functions are deterministic: using the same search criteria results in the same results.
```

Comment:
I think this is a bit misleading. I would change it for a definition of deterministic search.